src/django/pfb_analysis/models.py

```diff
@@ -557,8 +557,11 @@ def destinations_urls(self):
     def tile_urls(self):
         return [{
             'name': layer,
-            'url': self._s3_url_for_result_resource('tiles/neighborhood_{}'.format(layer) +
-                                                    '/{z}/{x}/{y}.png')
+            'url': (
+                settings.TILEGARDEN_ROOT +
+                '/tile/{job_id}/{layer}/'.format(job_id=self.uuid, layer=layer) +
+                '{z}/{x}/{y}.png'
```

Comment:
Is there a reason this can't go in the string above? Can the brackets not be escaped?

Reply:
What about
```python
'url': '{root}/tile/{job_id}/{layer}/{tile_template}'.format(
    root=settings.TILEGARDEN_ROOT,
    job_id=self.uuid,
    layer=layer,
    tile_template='{z}/{x}/{y}.png',
)
```
?

I find the escaped brackets very unsightly
```python
'url': '{root}/tile/{job_id}/{layer}/\{z\}/\{x\}/\{y\}.png'.format(
    root_url=settings.TILEGARDEN_ROOT, job_id=self.uuid, layer=layer)
```

Reply:
That does look nice